src/api/app/datatables/project_datatable.rb

```diff
@@ -22,9 +24,13 @@ def projects
   # rubocop:disable Naming/AccessorMethodName
   def get_raw_records
     if projects
-      projects
+      projects.left_joins(label_globals: [:label_template_global]).references(:label_globals, :label_template_global).distinct
```

Comment:
```suggestion
      project_list = projects || Project

    project_list = project_list.left_joins(label_globals: [:label_template_global]).references(:label_globals, :label_template_global)
    project_list = project_list.filtered_for_list unless show_all

    project_list.distinct
```

what do you think about a simpler refactoring like this?

Reply:
will follow up on this :)